skbio/alignment/_tabular_msa.py

```diff
@@ -570,3 +549,61 @@ def reindex(self, key=None, keys=None):
             keys_.flags.writeable = False
 
         self._keys = keys_
+
+    @experimental(as_of='0.4.0-dev')
+    def append(self, sequence):
+        """Append a sequence to the MSA.
+
+        Parameters
+        ----------
+        sequence : IUPACSequence
```

Comment:
Can you change the type to be `alphabet-aware scikit-bio sequence object`? We're about to change the inheritance hierarchy a bit to accommodate custom sequence classes that aren't IUPAC.